Guide-‐-Event-Manager.md

```diff
@@ -164,17 +177,26 @@ We use `timer = "UPTIME"` here as the default timer (`"REAL"`) is reset when ret
 
 This event also sets some properties to make sure it doesn't get blocked or block other events, and it runs even if the game is paused. If you want to make sure you can stop this event, then you could, say, add some check for some variable then return if it's true.
 
-
-## More properties
-These are some more event properties you can use, but aren't really that useful in most situations.
-
-start_timer - boolean: If true, then it will use the starting time of the event being created. Otherwise this will start the timer the first time the event is processed. Default is false.
-
-no_delete - boolean: If true, then when clearing the event queue, this event will not be deleted. You probably don't want to use this unless you want to have an event that never stops. Default is false.
-
-force_pause - boolean: If true, then the event will run even if the game is paused. This only has an effect if your event was created when the game wasn't paused. Default is false.
-
-timer - string: The name of the timer to use. Default is "REAL" if created while paused (or with force_pause) otherwise "TOTAL". Can take any of the keys in G.TIMERS.
+Here's another way to accomplish the same thing:
+```lua
+local create_event
+create_event = function()
+    G.E_MANAGER:add_event(Event {
```

Comment:
Might be worth adding the front argument in this case.

Reply:
Why? The event has `blockable = false` anyways. I'd much rather use an example where `front` is actually necessary but I genuinely can't think of anything.

Reply:
Good point. Forgot about that.